Innohoot/DTO/PollCollectionDTO.cs

```diff
@@ -7,6 +7,6 @@ public class PollCollectionDTO
 		public string Name { get; set; }
 		public string? Description { get; set; }
 		public DateTime Created { get; set; }
-		public List<Guid> Polls { get; set; }
+		public List<PollDTO> Polls { get; set; }
```

Comment:
Yes, It better to have PollDTO inside of PollCollectionDTO and OptionDTO inside of PollDTO